src/FormBuilder.php

```
@@ -88,23 +88,23 @@ public function submit(): array
      */
-    public function text(string $label, string $placeholder = '', string $default = '', bool|string $required = false, mixed $validate = null, string $hint = '', ?string $name = null, ?Closure $transform = null): self
+    public function text(string $label, string $placeholder = '', string $default = '', bool|string $required = false, mixed $validate = null, string $hint = '', ?string $name = null, ?Closure $transform = null, ?string $description = null): self
     {
         return $this->runPrompt(text(...), get_defined_vars());
     }
 
     /**
      * Prompt the user for multiline text input.
      */
-    public function textarea(string $label, string $placeholder = '', string $default = '', bool|string $required = false, ?Closure $validate = null, string $hint = '', int $rows = 5, ?string $name = null, ?Closure $transform = null): self
+    public function textarea(string $label, string $placeholder = '', string $default = '', bool|string $required = false, ?Closure $validate = null, string $hint = '', int $rows = 5, ?string $name = null, ?Closure $transform = null, ?string $description = null): self
```

PHPMD (2.15.0)

Avoid unused parameters such as '$label'. (Unused Code Rules)

(UnusedFormalParameter)